### DISCUSSION TITLE
Turbo requires a "packageManager" when it already has one

### DISCUSSION THREAD

**Discussion by fr0staman** [vercel/turborepo#7870] [Help] [answered]
### Summary

Hi!
I try integrate turbo in my monorepo.
So, `turbo run build` successfully works in my machine, but when build on Vercel, for some reason an error occurs (build error below)

But `"packageManager": "yarn@4.1.0"`, already exist in my root `package.json`, see repo example link below.

### Additional information

```js
x could not resolve workspaces: We did not find a package manager
  | specified in your root package.json. Please set the "packageManager"
  | property in your root package.json (https://nodejs.org/api/
  | packages.html#packagemanager) or run `npx @turbo/codemod add-package-
  | manager` in the root of your monorepo.
  `-> We did not find a package manager specified in your root package.json.
      Please set the "packageManager" property in your root package.json
      (https://nodejs.org/api/packages.html#packagemanager) or run `npx
      @turbo/codemod add-package-manager` in the root of your monorepo.
Error: Command "turbo run build" exited with 1
```

### Example

https://github.com/fr0staman/tamagui-react-native-i18next-zustand-starter

**Comment by anthonyshew** [comment]
What are your settings on your Vercel project?

**Reply by fr0staman** [reply]
Only stock settings, nothing has been changed
![image](https://github.com/vercel/turbo/assets/77665176/5afc583f-528d-47ee-8c86-a72d4a323ee6)

**Reply by fr0staman** [reply]
Any help?

**Reply by anthonyshew** [reply]
Sorry! GitHub notifications are terrible.

According to this page Yarn 4 does not appear to be supported on Vercel: https://vercel.com/docs/deployments/builds/package-managers#supported-package-managers

**Reply by fr0staman** [reply, marked as answer]
Thanks for trying!

Currently, I solved this problem by...

Overriding `turbo run build` to my standard build.
So, it works.
I don't downgraded yarn, just disabled turbo.

I'll be looking forward to improved support for Yarn v4!
